src/app/components/Playground.js

```diff
@@ -0,0 +1,17 @@
+class Playground {
+  constructor(id) {
+    this.render(id);
+  }
+
+  contentRender(id) {
+    const playground = document.querySelector(`#${id}`);
+    playground.classList.add('playground');
+
+  }
+
+  render(id) {
+    this.contentRender(id);
+  }
+}
+
+export default Playground;
```

Comment:
Czy potrzebujemy to jako komponent?

Reply:
aby aplikacja działała to nie potrzebujemy, ale tak zrozumiałem z treści taska, zdecyduj, jeśli mam to zrobić bez tego to tak zrobię

Reply:
Według mnie nie jest potrzebne ale zostawmy, może w przyszłości się przyda a pewnie jak już skończymy aplikacje to zrobimy mały refactoring i wtedy to najwyzej wyrzucimy